src/util/getStripeClient.ts

```diff
@@ -16,11 +14,5 @@ export function getStripeClient({
     ...options,
```

Comment:
Can you default the `appInfo` to the same values as before in the new format? Otherwise this looks great 👍